server/routes/api/sockets.js

```diff
@@ -0,0 +1,18 @@
+const router = require("express").Router();
+const cache = require("../../services/cache");
+const { v4: uuid } = require('uuid');
+
+router.post("/", async (req, res, next) => {
```

Comment:
This is a POST endpoint, any reason why this would be a POST rather than a GET?

Reply:
I thought I would be more like a login endpoint yet the auth middleware does all the work, so it makes sense to change it